File: client-app/src/app/models/entities/film.ts
import { User } from "../identity/user";

export interface Film{
    id: string,
    userId: string,
    user: User,
    title: string,
    genre: string,
    director: string,
    releaseYear: number,
    rating: number,
    description: string,
}
export interface FilmDto{
    id: string,
    userId: string,
    title: string,
    genre: string,
    director: string,
    releaseYear: number,
    rating: number,
    description: string,
}

File: Core/Domain/Entities/Film.cs
using Core.Domain.IdentityEntities;

namespace Core.Domain.Entities
{
    public class Film
    {
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public ApplicationUser? User { get; set; }
        public string Title { get; set; }
        public string Genre { get; set; }
        public string Director { get; set; }
        public int ReleaseYear { get; set; }
        public int Rating { get; set; }
        public string Description { get; set; }

    }
}